packages/bento-design-system/src/Illustrations/IllustrationProps.ts

```diff
@@ -1,11 +1,22 @@
+// NOTE(gabro): we deprecated the `style` prop due to false positive warnings in
+// popular ESLint configs, which expect `style` to be an object.
+// We're still keeping the old props around for backwards compatibility.
 export type IllustrationProps = {
   size: 24 | 32 | 40 | 80 | 160 | { custom: number };
 } & (
   | {
-      style: "color";
+      kind: "color";
```

Comment:
isn't it a breaking change in this way? I mean, the `kind` prop looks mandatory, so one has to add the kind everywhere, so what's the point in keeping also the style?

Reply:
Yeah, it was broken, fixed it in my last commit!